lib/dependencies/HarmonyExportExpressionDependency.js

```diff
@@ -121,7 +121,7 @@ HarmonyExportExpressionDependency.Template = class HarmonyExportDependencyTempla
 				runtimeRequirements.add(RuntimeGlobals.exports);
 				if (runtimeTemplate.supportsConst()) {
 					const name = "__WEBPACK_DEFAULT_EXPORT__";
-					content = `/* harmony default export */ const ${name} = `;
+					content = `/* harmony default export */ var ${name} = `;
```

Comment:
this works as expected since this `content` assignment is guarded by `runtimeTemplate.supportsConst()`

Reply:
The bundle js file will also contain const. Using babel has no effect. How to deal with this situation?